81-IDevID_and_IAK_support.md

```diff
@@ -173,8 +173,16 @@ bogged down.
 -->
 
 #### Story 1
+<!-- TODO: add IDevID scenario with Keylime using it to make sure the device is authentic. -->
+In order to securely identify that a device is genuine from a given OEM the IDevID can play an important role.
```

Comment:
I think this can be made stronger and clearer:
"The IDevID can plays an important role to securely identify that a device is genuine from a given OEM ."

This is vital for us. "*can play..*" implies there are other ways of doing it. I don't know that there are.

Reply:
agreed, "can play" was replaced by "plays" at the new phrasing.